src/Synchronization/OrderSync.php

```diff
@@ -73,7 +74,7 @@ public function sync($order_id ) {
                             'orderNumber' => $order->get_order_number(),
                             'shippingAddress' => $this->getShippingAddress($order),
                             'billingAddress' => $this->getBillingAddress($order),
-                            'status' => ['name' => $order->get_status()],
+                            'orderStatus' => ['code' => $this->getTrackMageStatus($order)],
```

Comment:
add `title` here. maybe `'orderStatus' => $this->getTrackMageStatus()`, and return ['code', 'title'] in `getTrackMageStatus`.